db/migrate/20191029210303_create_auctions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAuctions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :auctions do |t|
+      t.string :title
+      t.string :description
+      t.decimal :start_bid, precision: 10, scale: 2
+      t.decimal :current_bid, precision: 10, scale: 2
+      t.binary :image
+      t.datetime :end_time
```

Comment:
Could we rename this column to be more consistent with `created_at and `updated_at`?